common/dynamicconfig/constants.go

```diff
@@ -1058,6 +1058,30 @@ Note: this should be greater than matching.longPollExpirationInterval and matchi
 		defaultNumTaskQueuePartitions,
 		`MatchingNumTaskqueueReadPartitions is the number of read partitions for a task queue`,
 	)
+	MatchingBreakdownMetricsByTaskQueue = NewTaskQueueBoolSetting(
```

Comment:
since we usually need these all at the same time, how about a string setting that can contain a subset of `t`, `p`, and `b` characters?